packages/ui/core-components/src/lib/atoms/inputs/dropdown/stories/Dropdown.stories.svelte

```diff
@@ -12,8 +12,12 @@
 	import { query } from '@evidence-dev/universal-sql/client-duckdb';
 	import DropdownOption from '../helpers/DropdownOption.svelte';
 	import DependentDropdowns from './DependentDropdowns.story.svelte';
-
+	import DataTable from '../../../../unsorted/viz/table/DataTable.svelte';
```

Comment:
should I remove this from dropdown stories? i personally like having datatables to visually see the data being used/filtered

Reply:
Yes, please remove it. The stories should generally have only the component the story is for.